# main
verbose_main: false
main_iter_rate: 40
height: 2
look_ahead: 1
close_enough: 0.2
average_speed: 2.0

# motion
verbose_motion: true
use_ground_truth: true
qa: 1
qx: 1
qy: 1
qz: 1
enable_gps: true
r_gps_x: 0.019543745678429
r_gps_y: 0.030147210060821
r_gps_z: 0.0179255983443
enable_magnet: true
r_mgn_a: 0.00077211251238
enable_baro: true
r_bar_z: 0.031883941851213
enable_sonar: true
r_snr_z: 2.67472816353313e-05
motion_iter_rate: 20

# move
enable_move: true
verbose_move: true
Kp_lin: 0.5
Ki_lin: 0.0
Kd_lin: 0.0
Kp_z: 1
Ki_z: 0
Kd_z: 0
yaw_rate: 0.5
max_lin_vel: 2
max_z_vel: 0.5
move_iter_rate: 40

# run param used automatically and should not be written